### PR TITLE
Add pomodoro feature

### PR DESCRIPTION
**Pull request by harleenquinzell**
Hi! I liked your project and since I do big commits instead of small ones, I will rarely see a big list of commits and it will make me feel like I'm not doing anything... 😁

I usually look at my work time by how many pomodoros I've completed throughout the day, so I though it would also be great to have them in the tiny-care-terminal, where I could also see my stats.

I made it configurable so it can be disabled if not used. The length for the focus and break timers are also configurable.

I'm not sure where it could be visually added, so I just made a small room for it next to the parrot. 

I also used some ES6 styles, since I saw you have a PR for reformatting to it.

What do you think?

screen shot 2017-04-23 at 16 37 20

### CODE REVIEW

**Comment by gauravchl** [comment]
@harleenquinzell Cool! I was also working on it :)
check it out - https://github.com/gauravchl/tiny-care-terminal/commits/pomodoro

commands:

```
p - Switch parrot to pomodoro mode
s - Start pomodoro
e - Stop pomodoro
u - update pomodoro default duration
```

pmodoro


Setting the pomodo break time was remaining to implement.

**Comment by kandelvijaya** [comment]
Great.

**Comment by harleenquinzell** [comment]
That's awesome @gauravchl!
Wish I have thought of using the parrot too. Looks great =]

File: README.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ lists the environment variables that you can copy in your `rc` files:
   - `TTC_APIKEYS` -- set this to false if you don't want to use Twitter API
   keys and want to scrape the tweets instead.
   - `TTC_UPDATE_INTERVAL`, set this to change the update frequency in minutes, default is 20 minutes.
+  - `TTC_POMODORO` -- set this to false if you don't want to use the pomodoro timer
```
Comment (Garbee) [inline README.md]: false should be the default with `true` to turn the feature on IMO.

Comment (harleenquinzell) [inline README.md]: Sure! Makes sense. I'll make the change later today =]

**Comment by michellehartman89** [comment]
I'm just a starter but it looks good to me.

**Comment by notwaldorf** [comment]
@harleenquinzell @gauravchl awww this looks awesome! Thanks for working on it! I would lean towards doing this in the parrot, so that it's less cluttered (too many boxes gets messy and kinda stressfull). What do you think?

**Comment by notwaldorf** [comment]
I think this would also work well with #67, where the parrot can just be reused for different features

**Comment by Garbee** [comment]
Also if your focusing more on Pomodoros over commits, then it may be neat to see if the commits  bar area could be replaced with Pomodoros when in use. People who use this may find it more useful than commit count anyways.

**Comment by gauravchl** [comment]
@harleenquinzell  @notwaldorf Sounds great 😃 
Here is the PR: https://github.com/notwaldorf/tiny-care-terminal/pull/73

**Comment by harleenquinzell** [comment]
Yeah, I think it makes sense. I didn't want to replace the parrot without your opinion on it first.
Cool! I'll move the timer thing to the parrot and the bar chart will replace the commits bars.
I'll make sure it's all under the TCC_POMODORO config.

**Comment by notwaldorf** [comment]
I'm going to close this PR now, since #73 was merged! Thanks for the original suggestion! <3

**Pull request closed without merging**